UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking
+{
+    using global::Unity.XR.CoreUtils;
+    using System;
+    using System.Collections.Generic;
+    using UnityEngine.InputSystem;
+    using UnityEngine.XR.Interaction.Toolkit.Inputs;
+
+    /// <summary>
+    /// Sample for allowing the game object that this script is attached to follow the user's eye gaze
+    /// at a given distance of "DefaultDistanceInMeters". 
+    /// </summary>
+    public class FollowEyeGaze : MonoBehaviour
+    {
+        [Tooltip("Display the game object along the eye gaze ray at a default distance (in meters).")]
+        [SerializeField]
+        private float _defaultDistanceInMeters = 2f;
+
+        [SerializeField] private Color _idleStateColor;
```

Comment:
Nit/Style.

Please do not start serialized fields with an underscore.  For example...

- _idleStateColor-> idelStateColor

Reply:
Also, it would be ideal if all public members had summary blocks, describing what the field does. And for serialized fields, add a "tooltip"